test/tools/schema_validator/validator.cc

```diff
@@ -0,0 +1,62 @@
+#include "test/tools/schema_validator/validator.h"
+
+#include "common/router/config_impl.h"
+
+#include "test/test_common/printers.h"
+
+#include "tclap/CmdLine.h"
+
+namespace Envoy {
+
+const std::string Schema::ROUTE = "route";
+
+const std::string& Schema::toString(Type type) {
+  switch (type) {
+  case Type::Route:
+    return ROUTE;
```

Comment:
Yes, as mentioned in the description the tool will handle cluster and listener schema validation in the next iteration.